policies-and-procedures/new-team-member-nominations.md

```diff
@@ -0,0 +1,26 @@
+# New team member nomination process
+
+Jellyfin's contributor team is a dynamic, ever growing set of contributors who assist in the running of the project. Since Jellyfin is a "do-ocracy", team members make up the backbone of the project, doing numerous jobs such as writing code, reviewing suggestions, moderating the community, and helping to troubleshoot and test.
+
+This document outlines the steps that should be taken to nominate and accept a new member into the team.
+
+## Process
+
+1. Anyone who is currently on the Jellyfin team (defined as a member of #jellyfin-support-team and/or the GitHub organization) may nominate a community member for inclusion in the team.
+
+1. Nominations should reflect the following 4 goals:
```

Comment:
Might want to fix the numbers for each of these, I'm not sure if Github will auto-correct them.

Reply:
This is a markdown thing, it does. Having `1., 1.` will auto-format.